internal/uvm/start.go

```diff
@@ -54,25 +53,12 @@ func parseLogrus(r io.Reader) {
 	}
 }
 
-func processOutput(l net.Listener, doneChan chan struct{}, handler OutputHandler) {
-	defer close(doneChan)
-
-	c, err := l.Accept()
```

Comment:
Why cant we add a simple `context cancellation` and wait on a `select` for the `accept` or the `cancel` and issue the cancel when we receive an exited event?